src/main.c

```diff
@@ -120,9 +120,9 @@ void NitroMain (void)
     while (TRUE) {
         sub_02000F30();
         sub_0200106C();
-        sub_02017B9C();
+        ReadKeypadAndTouchpad();
 
-        if ((gCoreSys.unk_38 & (PAD_BUTTON_START | PAD_BUTTON_SELECT | PAD_BUTTON_L | PAD_BUTTON_R)) == (PAD_BUTTON_START | PAD_BUTTON_SELECT | PAD_BUTTON_L | PAD_BUTTON_R)) {
+        if ((gCoreSys.heldKeysRaw & (PAD_BUTTON_START | PAD_BUTTON_SELECT | PAD_BUTTON_L | PAD_BUTTON_R)) == (PAD_BUTTON_START | PAD_BUTTON_SELECT | PAD_BUTTON_L | PAD_BUTTON_R)) {
```

Comment:
This is the soft-reset input combo for the DS. Could we label the bitmask as such?